### PR TITLE
(DOCUMENTATION 📚) list common AST node types

### PR DESCRIPTION
**Pull request by urish**
Making it easier for fellow developers to compose queries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,3 +74,16 @@ The following selectors are supported:
...
+
+* `Identifier` - any identifier (name of a function, class, variable, etc)
+* `IfStatement`, `ForStatement`, `WhileStatement`, `DoStatement` - control flow
+* `FunctionDeclaration`, `ClassDeclaration`, `ArrowFunction` - declarations
+* `VariableStatement` - var, const, let.
+* `ImportDeclaration` - any `import` statement
+* `StringLiteral` - any string
+* `TrueKeyword`, `FalseKeyword`, `NullKeyword`, `AnyKeyword` - various keywords
+* `CallExpression` - function call
+* `FirstLiteralToken` - any numeric constant
```
Comment (phenomnomnominal) [inline README.md]: `FirstLiteralToken` is actually an alias for `NumericLiteral`

File: README.md [hunk 2]
```diff
@@ -74,3 +74,16 @@ The following selectors are supported:
...
+* `FirstTemplateToken`, `TemplateExpression` - template strings and expressions
```
Comment (phenomnomnominal) [inline README.md]: and `FirstTemplateToken` is an alias for `NoSubstitutionTemplateLiteral`

**Comment by urish** [comment]
Yes, totally. there are a bunch of others as well, such as `FirstBinaryOperator` which comes in place of `LessThanToken`, `FirstAssignment`, `LastKeyword`, etc.

**Comment by phenomnomnominal** [comment]
I'm gonna merge it for now, as it's "expected behaviour" and doesn't seem likely to change any time soon!

**Pull request merged**